.github/2.0/cookbooks/Executor.md

```diff
@@ -489,3 +489,33 @@ Line number corresponds to the 1.x code:
     - replacing previous `Blob2PngURI` and `ExcludeQL` driver logic using `Document` built-in
       methods `convert_blob_to_uri` and `pop`
     - there is nothing to return, as the change is done in-place.
+
+
+### Fastai
+
+
+```python
+import torch
+from jina import Executor, requests, DocumentArray, Document
+
+
+class ResnetImageEncoder(Executor):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        from fastai.vision.models import resnet18
+        
+        self.model = resnet18()
+
+     @requests
+     def encode(self, docs, **kwargs):
+         batch_images = docs.get_attributes('blob')
+         batch = Torch.Tensor(batch_images)
+         batch_embeddings = self.model(batch)
```

Comment:
this does not make much sense.  Shouldn't this be
`
batch_embeddings = self.model(batch).detach().numpy()
for doc, embedding in zip(docs, batch_embeddings)

`